test/js/comments.test.js

```diff
@@ -159,6 +159,16 @@ describe("comments", () => {
 
     test("command calls", () =>
       expect("command.call 'foo' # this is an inline comment").toMatchFormat());
+
+    test("chained command calls with comments", () => {
+      const content = ruby(`
+        puts foo
+          .bar # comment
+          .baz
+      `);
+
+      return expect(content).not.toFailFormat();
```

Comment:
I'm a little on the fence about this matcher. I could change this to `.toChangeFormat`, but that result isn't what I want. I could skip the test, but I want to ensure there's no regressions. Therefore I ended up with `.not.toFailFormat()` as it seems to be the best solution for now 🤔 

For visibility, the result is currently:

```
puts foo.bar.baz # comment
```